templates/patterns/list_item/pattern-list-item--variant-navigation.html.twig

```diff
@@ -4,10 +4,12 @@
  * Default implementation for the navigation list item component.
  */
 #}
-{% include '@ecl/ec-component-list-item' with {
-  'variant': 'navigation',
-  'title': title,
-  'href': url,
-  'detail': detail,
-  'screen_reader': 'List item'|t
+{% include '@ecl/card' with {
+  card : {
+    title: {
+      label: title,
+      path: url
+    },
+    description: detail
+  }
```

Comment:
This should also be done with content item composition, we use the card only on the highlight variant.

Reply:
ok